### PR TITLE
[REF] mrp: byproduct 

### PR DESCRIPTION
**Pull request by amoyaux**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mrp/views/mrp_workorder_views.xml [hunk 1]
```diff
@@ -166,6 +166,18 @@
                             <field name="move_id" invisible="1"/>
                         </tree>
                     </field>
+                    <h4 attrs="{'invisible': [('finished_workorder_line_ids', '=', [])]}">Finished Products</h4>
```
Comment (Whenrow) [inline addons/mrp/views/mrp_workorder_views.xml]: 'ByProduct' ?

File: addons/mrp/models/stock_move.py [hunk 2]
```diff
@@ -71,7 +71,7 @@ class StockMove(models.Model):
     # Quantities to process, in normalized UoMs
     bom_line_id = fields.Many2one('mrp.bom.line', 'BoM Line')
     byproduct_id = fields.Many2one(
-        'mrp.bom.byproduct', 'Subproduct',
+        'mrp.bom.byproduct', 'Byproduct',
         help="Subproduct line that generated the move in a manufacturing order")
```
Comment (sle-odoo) [inline addons/mrp/models/stock_move.py]: help should also use byproduct right?

File: addons/mrp/models/mrp_abstract_workorder.py [hunk 3]
```diff
@@ -20,13 +20,22 @@ class MrpAbstractWorkorder(models.AbstractModel):
     final_lot_id = fields.Many2one('stock.production.lot', string='Lot/Serial Number', domain="[('product_id', '=', product_id)]")
     product_tracking = fields.Selection(related="product_id.tracking")
 
+    @api.depends('raw_workorder_line_ids', 'finished_workorder_line_ids')
+    def _compute_workorder_line_ids(self):
+        for workorder in self:
+            workorder.workorder_line_ids = workorder.raw_workorder_line_ids | workorder.finished_workorder_line_ids
```
Comment (sle-odoo) [inline addons/mrp/models/mrp_abstract_workorder.py]: i don't think we need a compute here, an helper would be enough

**Comment by amoyaux** [comment]
check PR https://github.com/odoo/odoo/pull/32540

**Pull request closed without merging**